api/src/main/java/org/openmrs/module/attachments/AttachmentsServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package org.openmrs.module.attachments;
+
+import org.openmrs.*;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.attachments.obs.Attachment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AttachmentsServiceImpl implements AttachmentsService {
+	
+	@Override
+	public List<Attachment> getAttachments(Patient patient, Visit visit, Encounter encounter, boolean includeRetired) {
+		
+		List<Obs> obs = Context.getObsService().getObservationsByPerson(patient);
```

Comment:
Use [this](https://github.com/openmrs/openmrs-core/blob/33d7ebca048b1a5b32a84b008cf8d91cde8986e1/api/src/main/java/org/openmrs/api/ObsService.java#L231-L233) method.